x-pack/legacy/plugins/monitoring/public/np_imports/angular/index.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import angular, { IModule } from 'angular';
+
+import { AppMountContext, LegacyCoreStart } from 'kibana/public';
+
+// @ts-ignore TODO: change to absolute path
+import uiRoutes from 'plugins/monitoring/np_imports/ui/routes';
+// @ts-ignore TODO: change to absolute path
+import { uiModules } from 'plugins/monitoring/np_imports/ui/modules';
+// @ts-ignore TODO: change to absolute path
+import { registerTimefilterWithGlobalState } from 'plugins/monitoring/np_imports/ui/timefilter';
+import { configureAppAngularModule } from '../legacy_imports';
+
+import { localAppModule, appModuleName } from './modules';
+
+export class AngularApp {
+  private injector?: angular.auto.IInjectorService;
+
+  constructor(
+    { core }: AppMountContext,
+    { appBasePath, element }: { appBasePath: string; element: HTMLElement }
+  ) {
+    uiModules.addToModule();
```

Comment:
Yeah to avoid making a lot of changes to our "legacy" code I decided to collect the modules first and then apply them into our local angular. This way we don't have to make a lot of changes to files that use `uiModules` which can maybe help us preserve some unit-tests et. This also allocates all the changes into a new directory which will be easy to follow